docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,286 @@
+- [Theming bottom sheets](#theming-bottom-sheets)
+
+## Using bottom sheets
+
+Bottom sheets are supplementary surfaces primarily used on mobile.
+
+Before you can use bottom sheets, you need to import the Material Components package for Flutter:
+
+```dart
+package:flutter/material.dart
+```
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+### Making bottom sheets accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Types
+
+There are three types suitable for different use cases:
+1. [Standard bottom sheets](#standard-bottom-sheet) display content that complements the screen’s primary content and remain visible while users interact with the primary content.
+1. [Modal bottom sheets](#modal-bottom-sheet) are an alternative to inline menus or simple dialogs on mobile and provide room for additional items, longer descriptions, and iconography, and must be dismissed in order to interact with the underlying content.
+1. [Expanding bottom sheets](#expanding-bottom-sheet) provide a small, collapsed surface that can be expanded by the user to access a key feature or task to offer the persistent access of a standard sheet with the space and focus of a modal sheet.
+
+![Composite image of bottom sheet types](assets/bottom_sheets/bottom-sheet_types_composite.png)
+
+## Standard bottom sheet
+
+Standard bottom sheets co-exist with the screen’s main UI region and allow for simultaneously viewing and interacting with both regions. They are commonly used to keep a feature or secondary content visible on screen when content in main UI region is frequently scrolled or panned.
+
+### Standard bottom sheet example
+
+`BottomSheet`
+
+- [Class definition](https://api.flutter.dev/flutter/material/ScaffoldState/showBottomSheet.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/bottom_sheet.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/bottom_sheets/standard&gh_ref=develop)
+
+
+The following is an example expanded standard bottom sheet:
+![An expanded standard bottom sheet with header and list items](assets/bottom_sheets/SheetsBottom_standard_screenshot.jpg)
```

Comment:
Suggested alt-text:

Bottom sheet with "Header" and 5 items partially covering image of paper stacks.